lib/spack/spack/environment.py

```diff
@@ -71,6 +77,11 @@
 #: legal first keys in the spack.yaml manifest file
 env_schema_keys = ('spack', 'env')
 
+# Magic names
+user_speclist_name = 'specs'
+def_view_name = 'default'
```

Comment:
Rename: 
- `def_view_name` -> `default_view_name`
- `def_view_link` -> `default_view_link`

Also the comment "magic names" isn't the greatest comment.    I'd put a descriptive comment on each of these.

Reply:
Ok, but fair warning it's going to result in really ugly shenanigans to make lines fit under the character limit later. That's why I shortened them.

Reply:
It shouldn't.

Reply:
The shenanigans weren't as bad as I remembered, but it does hurt my soul to create a new variable just to have a shorter name.